Automattic/MobilePay-Apple
# Add: macOS example

Part of https://github.com/wordpress-mobile/WordPress-iOS/issues/16861

* Added a macOS example
* Changed the `MobilePayKit.framework` to be used for iOS and macOS

### To test

1. Add your token
2. Run the mac app
3. Make sure the product appears
4. Buy it :)
